plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/SnykReader.java

```diff
@@ -0,0 +1,112 @@
+package org.owasp.benchmarkutils.score.parsers;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.owasp.benchmarkutils.score.*;
```

Comment:
Please change wildcard import to concrete ones.

Reply:
Intellij is replacing the headers with a wildcard.